react/react1/week2/todos-app/src/components/AddTodoHandler.jsx

```diff
@@ -0,0 +1,8 @@
+export default function AppTodoBtn({ todo, setTodo }) {
+  const addTodoHandler = () => {
+    const newTodo = { id: Math.random() * 1000, description: "randomText" };
```

Comment:
Instead of using random for ids, research about the unique ids!

Reply:
Hi @Divyajg  

Thank you so much for your review! 
I have fixed my random ids and used nanoId instead now and als rename my mapTodo to renderTodos 

Pls let me know if I have to change something else or I can merge it? :)